test/EFCore.PG.FunctionalTests/Query/EnumQueryTest.cs

```diff
@@ -74,7 +74,7 @@ public void Where_with_parameter()
 
 SELECT s.""Id"", s.""EnumValue"", s.""InferredEnum"", s.""MappedEnum"", s.""SchemaQualifiedEnum"", s.""UnmappedEnum""
 FROM test.""SomeEntities"" AS s
-WHERE (s.""MappedEnum"" = @__sad_0) AND (@__sad_0 IS NOT NULL)
+WHERE s.""MappedEnum"" = @__sad_0
```

Comment:
Finally there is no trash in queries!

Reply:
There is still some trash :) but yeah, it's better :)